style.css

```diff
@@ -352,6 +365,121 @@ main {
   transform: scale(1.2);
 }
 
+/* MOBILE OVERLAY */
+.modal-mobile-overlay {
+  background-color: rgb(193, 199, 208, 80%);
+  display: none;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: 1rem 0;
+  position: absolute;
+  z-index: 10;
+  overflow-y: auto;
+  top: 0;
+}
```

Comment:
- [ ] Kindly give a gap between the two buttons so that it would look like the Figma.

### Your design
![image](https://user-images.githubusercontent.com/86200356/209179810-7a2085be-ad25-4b97-82f4-9dd8e24222ba.png)
### Figma
![image](https://user-images.githubusercontent.com/86200356/209179943-dbebc429-1df6-45db-a011-f57955af8742.png)

Reply:
Noted. I will fix this.